assets/section-main-product.css

```diff
@@ -651,6 +646,23 @@ a.product__text {
   border-radius: var(--media-radius);
 }
 
+.contains-media .grid_item .product__media-toggle:focus-visible:after {
+  box-shadow: 0 0 0 0.3rem rgb(var(--color-background)),0 0 0rem 0.5rem rgba(var(--color-foreground), 0.5);
+  border-top-left-radius: calc(var(--top-left-radius) - var(--media-border-width));
+  border-top-right-radius: calc(var(--top-right-radius) - var(--media-border-width));
+  border-bottom-right-radius: calc(var(--bottom-right-radius) - var(--media-border-width));
+  border-bottom-left-radius: calc(var(--bottom-left-radius) - var(--media-border-width));
+}
+
+.contains-media .grid_item .product__media-toggle.focused:after {
+  box-shadow: 0 0 0 0.3rem rgb(var(--color-background)),0 0 0rem 0.5rem rgba(var(--color-foreground), 0.5);
+  border-radius: var(--top-left-radius) var(--top-right-radius) var(--bottom-right-radius) var(--bottom-left-radius);
+}
+
+.contains-media .grid_item .product__media-toggle:focus-visible:after {
+  border-radius: var(--top-left-radius) var(--top-right-radius) var(--bottom-right-radius) var(--bottom-left-radius);
+}
+
```

Comment:
All this, as well as the original lines above will likely be removed with the upcoming focus style refactor. For now, this must be included to not introduce regressions to the product media focus outline